src/plugins/event-plugins/__tests__/TTIPlugin.test.ts

```diff
@@ -0,0 +1,124 @@
+import { TTIBoomerang } from '../../../time-to-interactive/TTIBoomerang';
+import { mockLongTaskPerformanceObserver } from '../../../test-utils/mock-data';
+import { TTIPlugin } from '../TTIPlugin';
+import { context, record } from '../../../test-utils/test-utils';
+import { TIME_TO_INTERACTIVE_EVENT_TYPE } from '../../../plugins/utils/constant';
+
+describe('Time to Interactive - Plugin Tests', () => {
```

Comment:
issue: I don't see a test that verifies the TTI value is correct. We could add one here, or in the integ test.

Reply:
Is it possible to validate the TTI value other than a assertNotNull? I'm not sure how we would could reproduce the quiet window

Reply:
in integ test will be hard to predict the value due to too many variables.
I believe the approach would be to mock TimeToInteractive's instance variables and then verify the functions provide the right values.

Reply:
> I believe the approach would be to mock TimeToInteractive's instance variables

Yes, something like this.

The unit test coverage of *TTIPlugin.ts* looks good to me. However, we've implemented the TTI logic in *TimeToInteractive.ts*, which does not have unit tests. Unit tests for *TimeToInteractive.ts* should go in `src/time-to-interactive/__tests__/TimeToInteractive.test.ts`.

Reply:
added some tests to exercise behavior of _TimeToInteractive.ts_